File: spring-boot-rest-services-java/src/main/java/de/larmic/rest/rest/InMemoryRepository.java
package de.larmic.rest.rest;

import de.larmic.rest.rest.dto.TweetDto;
import org.springframework.stereotype.Repository;

import java.util.HashMap;
import java.util.Map;

@Repository
public class InMemoryRepository {

    public final Map<String, TweetDto> tweets = new HashMap<>();

}

File: spring-boot-rest-services-kotlin/src/main/kotlin/de/larmic/rest/rest/InMemoryRepository.kt
package de.larmic.rest.rest

import de.larmic.rest.rest.dto.TweetDto
import org.springframework.stereotype.Repository

@Repository
class InMemoryRepository(val tweets: MutableMap<String, TweetDto> = mutableMapOf()) {

    fun exists(id: String) = tweets.containsKey(id)

}